examples/ota-requestor-app/linux/main.cpp

```diff
@@ -49,6 +52,11 @@ using namespace chip::ArgParser;
 using namespace chip::Messaging;
 using namespace chip::app::Clusters::OtaSoftwareUpdateProvider::Commands;
 
+OTARequestor requestorCore;
```

Comment:
Is this what was causing your ExchangeContext to go null? They go out of scope after main is called?

Reply:
I've run into issues in previous example app iterations where declaring a pointer in `main()` caused it to later go out of scope and get free'd and segfault in the middle of execution.

However, this was not what was causing my ExchangeContext issue